### PR TITLE
22 account management

### LINKED ISSUE #22: Account Manage Screen
Allows the user to change their name, email or password

### PR DESCRIPTION
**Pull request by MarshallAsch**
closes #22, This adds the ability to update a users first name, last name, email and password. 

Need to check to make sure that the font size on the general account info page is large / bold enough and if the buttons are big enough to press easily.

### CODE REVIEW

**Comment by weihanli101** [comment]
Sorry forgot to add this one. The attached screen shot would be an example of an icon that fits more into the theme of the app.


screen shot 2018-07-05 at 2 18 49 pm

**Comment by MarshallAsch** [comment]
The back button exiting from the dash, that was added quite a while ago and I do belive that is "normal" behaviour that we are not supposed to change according to googles pest practices. 
I will update the edit buttons on the account management page. For the edit button on the dashboard, Im not sure how to make that button but I will see what I can do.

File: app/src/main/java/ca/marshallasch/veil/utilities/PasswordState.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package ca.marshallasch.veil.utilities;
+
+/**
+     * This is returned from {@link #checkPasswords(String, String)} to determine what was wrong with the password
+     */
+public enum PasswordState
```
[the remark this answers is not in the document]
Comment (MarshallAsch) [reply app/src/main/java/ca/marshallasch/veil/utilities/PasswordState.java]: I think that we are going to need to refactor the util package, but I disagree on putting enums into its own package, we should look into other projects to see what they do tho.

**Comment by MarshallAsch** [comment]
Changes fixed in commit 3b2add1

**Pull request merged**